isdown/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "98d8eda3-7bc5-4067-8ff6-63d11d0132e2",
+  "app_id": "isdown",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "IsDown helps companies monitor all third-party status pages in one place",
+    "title": "IsDown",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "IsDown and Datadog Flow",
+        "image_url": "images/isdown_datadog_flow.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "IsDown Dashboards",
+        "image_url": "images/isdown_dashboards.jpg"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration"
```

Comment:
Please include categories listed [here](https://docs.datadoghq.com/integrations/)

Reply:
Hi @emarsha94, I've added the categories. 

Not entirely sure of the difference between Alerting and Notification.